$NetBSD: patch-CMakeLists.txt,v 1.1 2019/08/07 11:48:16 nia Exp $

Use the standard method to find OpenAL, it's more portable.

--- CMakeLists.txt.orig	2019-08-07 09:19:06.259170755 +0000
+++ CMakeLists.txt
@@ -753,7 +753,7 @@ find_package(SDL2_image REQUIRED)
 find_package(SDL2_ttf REQUIRED)
 find_package(Boost 1.33.1 COMPONENTS system filesystem)
 find_package(PNG REQUIRED)
-find_package(OpenALSoft REQUIRED)
+find_package(OpenAL REQUIRED)
 find_package(TinyXML REQUIRED)
 find_package(OGG REQUIRED)
 find_package(VORBIS REQUIRED)
@@ -876,7 +876,7 @@ include_directories(${SDL2_INCLUDE_DIR}
                     ${SDL2_TTF_INCLUDE_DIR})
 include_directories(${Boost_INCLUDE_DIRS})
 include_directories(${PNG_INCLUDE_DIRS})
-include_directories(${OPENALSOFT_INCLUDE_DIR})
+include_directories(${OPENAL_INCLUDE_DIR})
 include_directories(${OGG_INCLUDE_DIRS})
 include_directories(${VORBIS_INCLUDE_DIRS})
 include_directories(${TinyXML_INCLUDE_DIRS})
@@ -920,7 +920,7 @@ if(build-python)
   swig_link_libraries(fife ${SDL2_LIBRARY}
                            ${SDL2_IMAGE_LIBRARIES}
                            ${SDL2_TTF_LIBRARIES}
-                           ${OPENALSOFT_LIBRARY})
+                           ${OPENAL_LIBRARY})
   swig_link_libraries(fife ${Boost_LIBRARIES})
   swig_link_libraries(fife ${PNG_LIBRARIES})
   swig_link_libraries(fife ${VORBISFILE_LIBRARY})
@@ -1022,7 +1022,7 @@ if(build-library)
   target_link_libraries(fife ${SDL2_LIBRARY}
                              ${SDL2_IMAGE_LIBRARIES}
                              ${SDL2_TTF_LIBRARIES}
-                             ${OPENALSOFT_LIBRARY})
+                             ${OPENAL_LIBRARY})
   target_link_libraries(fife ${Boost_LIBRARIES})
   target_link_libraries(fife ${PNG_LIBRARIES})
   target_link_libraries(fife ${VORBISFILE_LIBRARY})
